### DISCUSSION TITLE
Dealing with multiple entries from the pickers of the Space mod

### DISCUSSION THREAD

**Discussion by dimus** [helix-editor/helix#10522] [Ideas]
Helix provides a powerful Space mod, which allows to search for files `Space->f`, global patterns `Space->/`, buffers `Space->b` etc.

Quite often more than one result of the search is correct, for example when there is a need to do something with a globally searched string.
It would be very useful to save all such entries into a list, which allows to traverse them one by one.

For example in Neovim/Telescope such behavior is achieved by filtering entries in Telescope and, when the desired subset of them is achieved, using `Ctrl-q` to transfer these entries to the quickfix window. These entries than can be traversed up and down with `]q` and `[q` "unimpaired" combinations.

I think such a feature would be very useful in many refactoring/editing scenarios.

The original idea for this came from a post by `Auke Roorda` in [Helix chat](https://matrix.to/#/!zMuVRxoqjyxyjSEBXc:matrix.org/$Itbj4-6aidVqZUuL86V7KIsoOzAXO-VYm9acpBejQuc?via=matrix.org&via=mozilla.org&via=envs.net).

**Comment by askreet** [comment]
Glad to hear others would find this useful. I have a _really_ basic implementation of this that I've been using on [this branch](https://github.com/askreet/helix/tree/quickfix), but I haven't thought through what is left to implement. One thing that's missing is a way to view the list, and another is keeping the list entries correct as files are updated (not even sure where to start with this one).
